Sources/IBMCloudAppID/Utils.swift

```diff
@@ -166,11 +166,10 @@ class Utils {
 
             /// The WebAppStrategy requires full token validation
             if options.shouldValidateAudAndIssuer {
-
-                guard token.aud == options.clientId else {
+                if !(token.aud?.contains(where: {$0 == options.clientId}) ?? true) {
```

Comment:
Doesn't this if statement always result in a "false"?

if `token.aud?.contains(where: {$0 == options.clientId})` == true -->` !true`
if `token.aud?.contains(where: {$0 == options.clientId})` == false then the coalesce `?? true` converts to true --> `!true` 

Maybe something like:

```suggestion
    guard let aud = token.aud, aud.contains { $0 == options.clientId } else {

```